docs/learning/tutorials/serializing-xml-schemas.md

```diff
@@ -0,0 +1,59 @@
+# Serializing EC Schemas to XML
+
+When working the [EC Schema instances](https://www.itwinjs.org/reference/ecschema-metadata/metadata/schema),
+it may be necessary to serialize the Schema to an XML file. The first option is a utility method found in the
+Bentley provided package [@itwin/ecschema-locaters](https://www.itwinjs.org/reference/ecschema-locaters).
+The second option utilizes two interfaces from the [@xmldom/xmldom](https://www.npmjs.com/package/@xmldom/xmldom)
+package. Both options will be covered here.
+
+## SchemaFileUtility in @itwin/ecschema-locaters
+
+The SchemaFileUtility class in the [@itwin/ecschema-locaters](https://www.itwinjs.org/reference/ecschema-locaters) package
+contains the 'writeSchemaXmlFile' method. This method takes the EC Schema instance and a path to the file to create. The
+SchemaFileUtility.writeSchemaXmlFile method is asynchronous, requiring the 'await' keyword.
+
+```ts
+await SchemaFileUtility.writeSchemaXmlFile(schema, outDir);
+```
+
+## @xmldom/xmldom package
+
+If you prefer to not have a dependency on the @itwin/ecschema-locaters package, the following example demonstrates how
+to serialize the Schema to an XML file using the DOMParser and XMLSerializer interface from the
+[@xmldom/xmldom](https://www.npmjs.com/package/@xmldom/xmldom) package. The [fs-extra](https://www.npmjs.com/package/fs-extra)
+package is used to work with the file system.
+
+```ts
```

Comment:
Please consider putting example code into the [snippets](https://github.com/iTwin/itwinjs-core/tree/master/example-code/snippets/src) package to ensure it compiles now and in the future.

Reply:
Thanks Paul.  Snippet created.